packages/http-client-python/eng/scripts/ci/regenerate.ts

```diff
@@ -222,16 +220,15 @@ async function executeCommand(tspCommand: TspCommand): Promise<void> {
   try {
     rmSync(tspCommand.outputDir, { recursive: true, force: true });
   } catch (error) {
-    console.error(`rm error: ${error}`);
+    console.error(chalk.red(`rm error: ${error}`));
   }
+  const execFileAsync = promisify(execFile);
   try {
-    console.log(`exec: ${tspCommand.command}`);
-    const { stdout, stderr } = await exec(tspCommand.command);
-    if (stdout) console.log(`stdout: ${stdout}`);
-    if (stderr) console.error(`stderr: ${stderr}`);
-  } catch (error) {
-    console.error(`exec error: ${error}`);
-    throw error;
+    await execFileAsync("tsp", tspCommand.command, { shell: true });
```

Comment:
Add `{shell: true}` to fix CI failure in Windows: https://dev.azure.com/azure-sdk/public/_build/results?buildId=4693056&view=logs&jobId=131ff7d7-c039-588f-c07d-15ff0819b235&j=131ff7d7-c039-588f-c07d-15ff0819b235&t=6739ba95-b8c8-5760-c562-c581774a89bd